examples/trivial/main.rs

```diff
@@ -15,13 +17,12 @@ use sirius::{
     ivc::{step_circuit, CircuitPublicParamsInput, PublicParams, IVC},
     poseidon::{self, ROPair},
 };
-use tracing::*;
 
 const ARITY: usize = BLOCK_SIZE / 2;
 
-const CIRCUIT_TABLE_SIZE1: usize = 16;
-const CIRCUIT_TABLE_SIZE2: usize = 16;
-const COMMITMENT_KEY_SIZE: usize = 19;
+const CIRCUIT_TABLE_SIZE1: usize = 20;
```

Comment:
do we have to increase to this level even for trivial example?

Reply:
In next PR